### DISCUSSION TITLE
Not able to access Education pack

### DISCUSSION THREAD

**Discussion by ayush222006** [community/community#77394] [GitHub Education] [answered]
### Hi everyone,

I have enrolled myself in education pack by using my student id.
and It got approved 
But I didn't receive any email about confirmation nor am I able to access the services 
I am a distance learner and used my personal email for it.

Please Help.

**Comment by CheetahCodes21** [comment, marked as answer]
It's understandable that you're eager to access the benefits of the GitHub Education Pack. While it's great that your application has been approved, it's possible that there might be a slight delay in receiving the confirmation email and accessing the services. This could be due to various factors, such as a high volume of applications or technical glitches.

To ensure you receive the confirmation email and access to the services, it's recommended to follow these steps:

1. **Check your spam folder:** Sometimes, emails from unfamiliar senders might end up in the spam folder. Check your spam folder to ensure the confirmation email hasn't been diverted there.

2. **Contact GitHub Education support:** If you haven't received the confirmation email even after checking your spam folder, reach out to GitHub Education support for assistance. They can help you verify your eligibility and expedite the process of receiving the confirmation email.

3. **Double-check your email address:** Verify that the email address you used for the GitHub Education Pack application is correct. If there's any typo or inconsistency, it could be preventing the confirmation email from reaching you.

4. **Check your GitHub account:** While the confirmation email might be delayed, you should still be able to access some of the GitHub Education Pack benefits, such as GitHub Actions for Students, through your GitHub account. Check your account settings to see if these benefits are available.

5. **Patience is key:** Sometimes, administrative processes can take a bit longer than expected. Be patient and continue to check your email and GitHub account for updates. If you haven't received the confirmation email or access to the services within a reasonable timeframe, contact GitHub Education support for further assistance.

Remember, GitHub Education is committed to providing students with access to valuable resources and tools. By following these steps, you should be able to access the benefits of the GitHub Education Pack soon.

**Comment by runrohit** [comment]
I am checking my account more than 3 times a day to see if I received the benefit. I am approved
